content/en/continuous_integration/pipelines/jenkins.md

```diff
@@ -1455,27 +1455,29 @@ Send pipeline traces.
 ...
 {{< /code-block >}}
 
-### The Datadog Plugin cannot write payloads to the server
+### Pipeline executions data is not available in Datadog
+
+#### HTTP connectivity check
+
+If your Jenkins instance is behind an HTTP proxy, go to **Manage Jenkins** > **Manage Plugins** > **Advanced tab** and make sure the proxy configuration is correct:
+- if the Datadog plugin is configured to send data to Datadog Agent, check that the agent host has been added to the `No Proxy Hosts` section
+- if the Datadog plugin is configured to send data directly to Datadog ("agentless mode"), check that Datadog host (`datadoghq.com` or `datadoghq.eu`, depending on which Datadog site you're using) has been added to the `No Proxy Hosts` section.
```

Comment:
```suggestion
- if the Datadog plugin is configured to send data directly to Datadog ("Agentless mode"), check the Datadog host (`datadoghq.com` or `datadoghq.eu`, depending on which Datadog site you're using) has been added to the `No Proxy Hosts` section.
```

Reply:
Is this only available for `US1` and `EU` sites?

Reply:
It's available for every site except DD Gov.
I didn't want to list every site explicitly to not be too verbose. All the US sites and the AP one are subdomains of `datadoghq.com` so I assumed including only the top-level domain would be enough.

Reply:
Thanks for clarifying. From past experience, this kind of omission can cause confusion and lead to support tickets. As such, I've requested to update this language and add a table of the supported sites and host values. I'm happy to keep iterating on this if you think the information could be presented more effectively.

Reply:
That makes sense, I agree that it's best to make things more explicit.